### DISCUSSION TITLE
如何实现思维导图中移动节点的功能？

### DISCUSSION THREAD

**Discussion by bigbigbo** [antvis/X6#2350] [Q&A]
现在有一个需求是需要实现一个思维导图的功能，并且需要支持移动节点到其他节点的功能，x6本身在节点有连线的情况下，移动节点那跟线也会跟着动，想实现像下图这样的效果如何实现呢？
![image](https://user-images.githubusercontent.com/19587016/178470015-385bcb89-7ca3-4678-aa64-bfaee333b634.png)

**Comment by DanielLeefu** [comment]
有一个思路可以尝试一下，就是移动当前节点的时候，拿到节点位置信息进行判断，当前节点位置靠近哪个目标节点，然后删除之前的连线重新创建和目标节点的连线
